src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectManagerService.cs

```diff
@@ -274,6 +274,30 @@ public async ValueTask<IProjectContextInfo> GetProjectAsync(string projectId, Ca
             return targetFrameworks;
         }
 
+        public async ValueTask<bool> IsCentralPackageManagementEnabledAsync(string projectId, CancellationToken cancellationToken)
```

Comment:
This one looks very similar to one in IProjectContextInfoExtensions.cs. Could they considerate into one?

Reply:
I looked at the code but don't see a clear way to make it simpler, do you see something here?